FHEM/00_SIGNALduino.pm

```diff
@@ -2366,32 +2386,23 @@ sub SIGNALduino_Parse_MU($$$$@)
 						
 					}
 					@bit_msg=(); # clear bit_msg array
+					last if ($lastSignal);
 					
 					#Find next position of valid signal (skip invalid pieces)
-					my $regex=".{$i}".$start_regex;
+					my $regex=".{$i}".$startStr.$signal_regex;
```

Comment:
Wir könnten hier doch einfach auch wieder in etwa so was machen und den Nächsten Start suchen:

`($rawData) = $rawData=~ /$startStr.*($startStr.*)$ `
Allerdings wenn wir keinen startStr haben, müsste das mit oder abgefragt werden, was es komplexer macht.

Reply:
Die zusammengebaute $regex müsste so aussehen oder?
https://regex101.com/r/vWXprk/1